package.json

```diff
@@ -41,10 +55,12 @@
     "@textile/threads-client-grpc": "0.0.1",
     "@types/google-protobuf": "^3.7.2",
     "google-protobuf": "^3.10.0",
-    "uuid": "^3.3.3"
+    "isomorphic-ws": "^4.0.1",
```

Comment:
Added these to support WebSockets in nodejs. Using this ensures our tests are isomorphic and can run in a browser once we get that going.

Reply:
Great!